File: testprojects/scala_main/pipelines/SCALA_BASIC_20_MINS/code/src/main/scala/com/scala/main/job1/graph/Subgraph_1/Subgraph_2/Reformat_5.scala
package com.scala.main.job1.graph.Subgraph_1.Subgraph_2

import io.prophecy.libs._
import com.scala.main.job1.udfs.UDFs._
import com.scala.main.job1.graph.Subgraph_1.Subgraph_2.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._
object Reformat_5 { def apply(context: Context, in: DataFrame): DataFrame = in }

File: testprojects/python_main/pipelines/PYTHON_UNITY_CATALOG/code/python_unity_catalog/graph/Reformat_5.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from python_unity_catalog.config.ConfigStore import *
from python_unity_catalog.udfs.UDFs import *

def Reformat_5(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0